app/views/building_map/index.html.erb

```diff
@@ -9,3 +9,46 @@
     polylines: BuildingMapHelper.leaflet_polylines(@route),
     markers: BuildingMapHelper.leaflet_markers(@route, @target))
 %>
+<style>
```

Comment:
Add a comment that explains this workaround

Reply:
Why is this a workaround? We are applying a style to a class, isn't this default CSS stuff?

Reply:
nevermind - because of all the `0` and `0px` stuff I thought this is a workaround to hide something